[2024-03-01T16:36:59.449+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T11:04:00+00:00 [queued]>
[2024-03-01T16:36:59.457+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T11:04:00+00:00 [queued]>
[2024-03-01T16:36:59.457+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-01T16:36:59.522+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_csvs_from_urls> on 2024-03-01 11:04:00+00:00
[2024-03-01T16:36:59.533+0530] {standard_task_runner.py:60} INFO - Started process 23125 to run task
[2024-03-01T16:36:59.539+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'datapipeline_Take4', 'fetch_csvs_from_urls', 'scheduled__2024-03-01T11:04:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/datapipeline_best(2).py', '--cfg-path', '/tmp/tmpcvl5g82q']
[2024-03-01T16:36:59.541+0530] {standard_task_runner.py:88} INFO - Job 135: Subtask fetch_csvs_from_urls
[2024-03-01T16:36:59.718+0530] {task_command.py:423} INFO - Running <TaskInstance: datapipeline_Take4.fetch_csvs_from_urls scheduled__2024-03-01T11:04:00+00:00 [running]> on host arun-VM
[2024-03-01T16:36:59.977+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='datapipeline_Take4' AIRFLOW_CTX_TASK_ID='fetch_csvs_from_urls' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T11:04:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T11:04:00+00:00'
[2024-03-01T16:37:00.969+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-03-01T16:37:00.984+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=datapipeline_Take4, task_id=fetch_csvs_from_urls, execution_date=20240301T110400, start_date=20240301T110659, end_date=20240301T110700
[2024-03-01T16:37:01.115+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T16:37:01.174+0530] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
